### DISCUSSION TITLE
Dredge (MuMIn) on constrained ordination not working?

### DISCUSSION THREAD

**Discussion by plbiella** [JenniNiku/gllvm#150] [Q&A]
Hi, I am trying a variable selection on a constrained ordination but it does not work; saying
**r in dredge(RRGLM, trace = 2, rank = "AICc") : result is empty**

the code is the following
----
data(spider)
Y <- spider$abund
X <- spider$x

#And scale the predictors
X <- scale(X)
RRGLM <- gllvm(Y, X = X, family = "poisson", num.RR = 2)

options(na.action = "na.fail")
fit_var <- dredge(RRGLM, trace=2, rank="AICc")

----

this returns immediately:
**r in dredge(RRGLM, trace = 2, rank = "AICc") : result is empty**

**Comment by BertvanderVeen** [comment]
Hello! gllvm is not supported by the MuMIn package, and although "dredge" can be conned into supporting some of the models in gllvm, the constrained ordination method is unfortunately not one of them yet (I think, but I am happy to be proven wrong). If you want to try different models with different numbers of predictors, you could try writing a loop instead?
